etc/git-extras-completion.zsh

```diff
@@ -441,3 +441,5 @@ zstyle ':completion:*:*:git:*' user-commands $existing_user_commands \
     undo:'remove latest commits' \
     unlock:'unlock a file excluded from version control' \
     utimes:'change files modification time to their last commit date'
```

Comment:
Missing a `\` at the end?

Reply:
Yes, you are correct. I missed that when I re-based.
I just updated the commit, and that is now fixed.